Web Development/README.md

```diff
@@ -221,6 +221,30 @@ This category is a curated collection of valuable web development resources cont
         <td> <a href="https://animista.net/play/basic/slide-bck">Animista</a></td>
         <td>Animista is a CSS animation library and a place where you can play with a collection of ready-made CSS animations.</td>
     </tr>
+    <tr>
+        <td> <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Animations">MDN Web Docs</a></td>
+        <td>MDN offers comprehensive guides and tutorials on web technologies, including CSS animations. Their documentation is reliable and up-to-date.</td>
+    </tr>
+    <tr>
+        <td> <a href="https://css-tricks.com/">CSS Tricks</a></td>
+        <td>CSS-Tricks is a website dedicated to CSS, and it has a wealth of articles and tutorials on various CSS topics, including animations.</td>
+    </tr>
+    <tr>
+        <td> <a href="https://www.w3schools.com/css/css3_animations.asp">W3Schools</a></td>
+        <td>W3Schools is a popular resource for web development tutorials, including CSS animations. It provides interactive examples and quizzes.</td>
+    </tr>
+    <tr>
+        <td> <a href="https://www.codecademy.com/learn/learn-css">Codecademy</a></td>
+        <td>Codecademy offers interactive coding lessons, including courses on CSS animations. It's a great platform for hands-on learning.</td>
+    </tr>
+    <tr>
+        <td> <a href="https://cssanimation.rocks/">CSS Animations</a></td>
+        <td>This site is dedicated to CSS animations and provides tutorials, demos, and resources for learning and mastering CSS animations.</td>
+    </tr>
+    <tr>
+        <td> <a href="https://animate.style/">Animate CSS</a></td>
+        <td>Animate.css is a popular library for adding ready-made CSS animations to your projects. You can learn from its examples and documentation.</td>
+    </tr>
```

Comment:
This is the only valid animation resource you added. Everything else moved it to the [tutorial or courses](https://github.com/jfmartinz/ResourceHub/tree/main/Web%20Development#tutorials-or-courses) section instead.

Example of a resource in animation section could bw something like [this](https://animista.net/play/basic/slide-bck)